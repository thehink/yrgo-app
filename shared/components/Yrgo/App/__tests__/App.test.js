/* eslint-disable import/no-extraneous-dependencies */

import React from 'react';
import { shallow } from 'enzyme';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';

const mockStore = configureStore();

import ConnectedApp, { App } from '../App';

describe('<App />', () => {
  const initialState = {};
  const store = mockStore(initialState);

  test('renders', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
