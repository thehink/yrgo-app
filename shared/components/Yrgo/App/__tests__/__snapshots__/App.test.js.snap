// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders 1`] = `
<app
  className="app"
>
  <HelmetWrapper
    encodeSpecialCharacters={true}
  >
    <title>
      Yrgo
    </title>
  </HelmetWrapper>
  <withRouter(Header) />
  <main
    className="undefined container-fluid"
  />
</app>
`;
